application/models/Kalkun_model.php

```diff
@@ -40,14 +40,37 @@ function login()
 		$this->db->where('username', $username);
 		$this->db->where('password', $password);
 		$query = $this->db->get();
-		
+
+		$this->session->set_flashdata('bef_login_history_count',
+            $this->session->flashdata('bef_login_history_count')-1);
+		$this->session->set_flashdata('bef_login_HTTP_REFERER', 
+            $this->session->flashdata('bef_login_HTTP_REFERER'));
+		$this->session->set_flashdata('bef_login_method', 
+            $this->session->flashdata('bef_login_method'));
+        $this->session->set_flashdata('bef_login_post_data',
+            $this->session->flashdata('bef_login_post_data'));
```

Comment:
Are you refereing to this ? Or also other parts ?

Reply:
Yes, and other places.

![image](https://user-images.githubusercontent.com/709368/73188518-01647400-4149-11ea-9f3c-5a0bb2631210.png)

The start of the lines are not indented properly.